ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -159,11 +179,14 @@ public NetworkClient(Selector selector, NetworkConfig networkConfig, int maxConn
    *                         the selector events.
    */
   private void handleSelectorEvents(List<ResponseInfo> responseInfoList) {
+    long handleSelectorEventsStartTimeMs = System.currentTimeMillis();
```

Comment:
What is the use of this metric? There are no I/O or cpu intensive stuff being done within this method.